lib/Subscribe.js

```diff
@@ -20,8 +21,43 @@ module.exports = async function(options = {
     try {
       let workerState = await subscribeWorker.getWorkerState(options.statePath)
       const result = await exec(`${__dirname}/subscribe-worker-batch.js ${workerState.statePath}`)
```

Comment:
consider adding a condition that checks for a --no-spawn option in the command and requires in the subscribe-worker-batch code - makes debugging easier.